server/models/HttpMethod.ts

```diff
@@ -4,3 +4,12 @@ export enum HttpMethod {
   PUT = "put",
   DELETE = "delete",
 }
+
+export function getDefaultStatusCodeForMethod(method: HttpMethod) {
```

Comment:
Default HTTP codes used by HTTP method if one if a `HttpResponse` is not returned by the RouteHandler